ramls/types/packages/packageRelationships.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Package relationships object schema in package collection item",
+  "description": "Package relationships object schema in package collection item",
+  "javaType": "org.folio.rest.jaxrs.model.PackageRelationship",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "resources": {
+      "type": "object",
+      "description": "Displays Resources Relationship if any",
+      "$ref": "../metaIncluded.json"
+    },
+    "vendor": {
```

Comment:
We can eliminate "vendor" from here. "vendor" was replaced by "provider" and probably, in some parts of Ruby code, we did not clean up "vendor" completely. UI only uses "provider" and has no knowledge of "vendor"